strings/ctype-czech.c

```diff
@@ -23,14 +23,14 @@
 	solution was needed than the one-to-one conversion table. To
 	note a few, here is an example of a Czech sorting sequence:
 
-		co < hlaska < hl�ska < hlava < chlapec < krtek
```

Comment:
All the "special" characters that are latin-1 encoded are actually in comments. Let's run iconv on this file and convert it to UTF-8. There should be no functional changes.

Reply:
The comments in this file have been changed to UTF-8